src/Board.js

```diff
@@ -11,7 +16,12 @@ function Board({ answer }) {
   const [error, setError] = useState("");
   const [message, setMessage] = useState("");
   const [gameOver, setGameOver] = useState(false);
-
+  const [gameWon, setGameWon] = useState(false);
```

Comment:
🎉  Clipboard paste works!!

Nit: Do we absolutely need a new state for `gameWon` or can we re-use the `gameOver` state?

Reply:
In the share message, there's a little 'X/6' on the top right. If the game is won, then X is a number. If the game is lost, then X is 'X'. There's probably a way to do this by checking if guess' length is less than 6, and then checking if the current guess is equal to the answer, but I feel keeping track of whether the game won or lost is useful information in any potential statistics panel.

Reply:
> keeping track of whether the game won or lost is useful information in any potential statistics panel.

I see, that works for me!